engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-details.html

```diff
@@ -38,6 +38,9 @@
                    ng-value="true"
                    type="radio" />
             Completely deletes the host record and removes all reporting, provisioning, and configuration information.
+            <p translate>
+            <b>Note:</b> While selecting the option <b>"Completely deletes the host record"</b>, if the Settings -> Katello -> <b>unregister_delete_host</b> option is set to <b>True</b>, the virtual machine will be deleted from the hypervisor.
```

Comment:
I don't think the `<b>Note:</b>` part is needed, the context makes it obvious it's a note.

Reply:
Also, the spacing here is off, this line should be indented by two.

Reply:
I'm also not a fan of the multiple `<b>`s in this line.